src/wristpy/core/models.py

```diff
@@ -18,10 +18,26 @@ class Config:
 
         arbitrary_types_allowed = True
 
-    @validator("time")
-    def validate_time(cls, v):  # noqa: ANN201, D102
+    @field_validator("time")
+    def validate_time(cls, v: pl.Series) -> pl.Series:
```

Comment:
Add a check to see if measurement/time are not empty. This can happen with actfast.